Packages/InputSystem/Runtime/CallbackContextEventVector2.cs

```diff
@@ -0,0 +1,7 @@
+using UnityAtoms.BaseAtoms;
+using UnityEngine;
+
+[CreateAssetMenu(menuName = "Unity Atoms/Examples/CallbackContext Vector2")]
+public class CallbackContextEventVector2 : CallbackContextEvent<Vector2, Vector2Pair, Vector2Constant, Vector2Variable, Vector2Event, Vector2PairEvent, Vector2Vector2Function, Vector2VariableInstancer>
```

Comment:
Rename it to `Vector2CallbackContextEvent` in order to follow the naming conventions of other atoms, eg. Variables.